azure-pipelines.yml

```diff
@@ -268,3 +268,17 @@ jobs:
       parameters:
         jobName: Correctness_Rebuild
         configuration: Release
+
+- job: Correctness_Analyzers
+  pool:
+    name: NetCore1ESPool-Public
+    demands: ImageOverride -equals Build.Windows.Amd64.VS2022.Pre.Open
+  timeoutInMinutes: 30
+  steps:
+    - template: eng/pipelines/checkout-task.yml
```

Comment:
Ah, looks like this changed in #61965

Reply:
`checkout-windows-task.yml` was removed due to the windows images no longer shipping with git installed. It will be restored in a later PR which moves us to using the git from the agent tools.